JPS_Ontology/ontology/ontobms/OntoBMS.csv

```diff
@@ -32,20 +32,22 @@ HeatDuty,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/HeatFlo
+hasAirFlowRate,Object Property,https://www.theworldavatar.com/kg/ontodevice/hasQuantity,IS-A,https://saref.etsi.org/core/HVAC,http://www.ontology-of-units-of-measure.org/resource/om-2/VolumetricFlowRate,,A relationship that describes the flow rate of air being processed or conveyed by a HVAC device.,https://www.theworldavatar.com/kg/ontobms/,has air flow rate
 conveys,Object Property,,,https://saref.etsi.org/saref4bldg/FlowMovingDevice,http://www.theworldavatar.com/ontology/ontocape/material/substance/substance.owl#Substance,,A relationship that describes the sustance being conveyed by a flow moving device (e.g. duct).,https://www.theworldavatar.com/kg/ontobms/,conveys
 hasLowLevelState,Object Property,,,https://saref.etsi.org/core/MultiLevelState,LowLevelState,,A relationship that describes a low level state of a multi-level state.,https://www.theworldavatar.com/kg/ontobms/,has low level state
 hasMediumLevelState,Object Property,,,https://saref.etsi.org/core/MultiLevelState,MediumLevelState,,A relationship that describes a medium level state of a multi-level state.,https://www.theworldavatar.com/kg/ontobms/,has medium level state
 hasHighLevelState,Object Property,,,https://saref.etsi.org/core/MultiLevelState,HighLevelState,,A relationship that describes a high level state of a multi-level state.,https://www.theworldavatar.com/kg/ontobms/,has high level state
+https://www.theworldavatar.com/kg/ontodevice/hasQuantity,Object Property,,,http://www.theworldavatar.com/ontology/ontocape/chemical_process_system/CPS_behavior/behavior.owl#MaterialAmount,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,,"A relationship between something (it could be a device or space, a setpoint, an operating property etc) and the quantity it represents or has.",https://www.theworldavatar.com/kg/ontobms/,has quantity
+https://www.theworldavatar.com/kg/ontodevice/hasQuantity,Object Property,,,http://www.theworldavatar.com/ontology/ontocape/chemical_process_system/CPS_function/process.owl#UnitOperation,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,,"A relationship between something (it could be a device or space, a setpoint, an operating property etc) and the quantity it represents or has.",https://www.theworldavatar.com/kg/ontobms/,has quantity
+https://www.theworldavatar.com/kg/ontodevice/hasQuantity,Object Property,,,http://www.theworldavatar.com/ontology/ontocape/material/phase_system/phase_system.owl#PhaseSystem,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,,"A relationship between something (it could be a device or space, a setpoint, an operating property etc) and the quantity it represents or has.",https://www.theworldavatar.com/kg/ontobms/,has quantity
```

Comment:
You're making a number of statements here about things in OntoDevice. I believe it would be better to move these to OntoDevice, rather to leave them here in OntoBMS...